### PR TITLE
setup notification request grpc

### PR DESCRIPTION
**Pull request by btkostner**

### CODE REVIEW

File: protos/bottle/account/v1.proto [hunk 1]
```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+import "bottle/account/v1/service.proto";
+
+package bottle.account;
+
+service V1 {
+  rpc NotificationMethod(bottle.account.v1.NotificationMethodRequest) returns (bottle.account.v1.NotificationMethodResponse);
```
Comment (doomspork) [inline protos/bottle/account/v1.proto:8]: I'm wonder if we should just return all of their notification preferences at once but we can deal with that later :+1:

**Review by doomspork — APPROVED** [review]

File: protos/bottle/account/v1/events.proto [hunk 2]
```diff
@@ -18,6 +18,11 @@ message PasswordReset {
   string reset_url = 2;
 }
 
+message TwoFactorAuthorizationRequested {
```
Comment (doomspork) [inline protos/bottle/account/v1/events.proto:21]: Do we need to add this to wrapper?

Comment (doomspork) [reply protos/bottle/account/v1/events.proto:21]: Also, do we want to shorten this to just `TwoFactorRequested`?

File: protos/bottle/account/v1.proto (hunk 1 above)
Comment (btkostner) [reply protos/bottle/account/v1.proto:8]: I think this is good for now. We can update it later. One thing that I was thinking was also maybe moving notification preferences into bullhorn and have _it_ keep track of everything, but I'd rather just get this to a working state first.
